general_considerations.md

```diff
@@ -152,6 +167,24 @@ times faster than bitcoin blocks, and results in miners 600 times smaller being
 able to contribute, and a 600x factor reduction in variance compared to solo
 mining.
 
+[kp] 
+
+There's a different way to think about this. You can reach consensus only after
+everyone has seen the same information, that is true. However, thinking in terms
+of eventual consistency, you don't have the time pressure anymore. This is
+enabled by and is required in the asynchronous communication model that we live
+in.
+
+All we want to do is achieve consensus on reward distribution for happened at
+time $T$ in $T + \delta T$ time.
+
+If miners starts broadcasting shares at time $T$ and the next block in found in
+$T + F$, then we need to reach consensus at some time in the future $T + F +
+\delta T$, s.t. $\delta T$ meets the conditions that 1) it is acceptable to
+miners; 2) doesn't violate the incentives compatible requirements.
+
+[/kp]
+
```

Comment:
Consensus in a braid is defined by a graph cut. We can't reason about timestamps with any reliability. If we could, we wouldn't need a consensus algorithm, just time-order everything!

The braid retargets its difficulty in effect definining $\delta F$.